global.json

```diff
@@ -1,6 +1,6 @@
 {
   "tools": {
-    "dotnet": "5.0.100-alpha1-014854"
+    "dotnet": "5.0.100-alpha1-015772"
```

Comment:
The latest daily sdk is `5.0.100-alpha1-016056` maybe worth updating to the latest?

Reply:
I used the value from https://github.com/dotnet/runtime/blob/8c0a1d91986f20d49ebfa07bd39ae60b9dd31394/global.json#L8 which presumably is a known-good one :)

This should only be needed for a few weeks until we get publishing up in dotnet/runtime.